source/php-frameworks/symfony.txt

```diff
@@ -0,0 +1,387 @@
+
+.. _php-symfony-qs:
+
+Quick Start
+-----------
+
+This guide shows you how to build a web application by using Symfony, a
+PHP framework. You will learn how to connect to a MongoDB cluster hosted
+on MongoDB Atlas and access data from your database.
+
+By using MongoDB as a data store in a Symfony web application, you can
+leverage the document data model to build rich query expressions,
+allowing you to easily interact with data.
+
+.. tip::
+
+   If you prefer to connect to MongoDB by using the MongoDB PHP Library without
+   Symfony, see `Connecting to MongoDB
+   <https://www.mongodb.com/docs/php-library/current/tutorial/connecting/>`__
+   in the MongoDB PHP Library documentation.
+
+MongoDB Atlas is a fully managed cloud database service that hosts your
+MongoDB deployments. You can create your own free (no credit card
+required) MongoDB Atlas deployment by following the steps in this guide.
+
+This guide uses MongoDB Doctrine ODM to allow you to map PHP objects to
+MongoDB documents and query MongoDB by using a builder API. 
+
+Follow the steps in this guide to create a sample Symfony web application
+that connects to a MongoDB deployment and performs a query on the
+database.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+To create the Quick Start application, you need the following software
+installed in your development environment:
+
+- `PHP <https://www.php.net/downloads>`__
+- `Composer <https://getcomposer.org/doc/00-intro.md>`__
+- `Symfony CLI <https://symfony.com/download>`__
+- A terminal app and shell. For MacOS users, use Terminal or a similar app.
+  For Windows users, use PowerShell.
+
+.. _php-symfony-atlas-cluster:
+
+Create a MongoDB Atlas Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You must create a MongoDB cluster where you can store and manage your
+data. Complete the
+:atlas:`Get Started with Atlas </getting-started>` guide to set up a new
+Atlas account, create a free tier MongoDB cluster, load datasets, and
+interact with the data.
+
+You provide instructions to the driver on where and how to connect to your
+MongoDB cluster by passing a string called the connection string. To retrieve
+your connection string, follow the instructions in the :atlas:`Connect
+to Your Cluster </tutorial/connect-to-your-cluster/>` tutorial in the
+Atlas documentation.
+
+Save your connection string in a secure location.
+
+Initialize a Symfony Project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following command to create a skeleton Symfony project called
+``restaurants``:
+
+.. code-block:: bash
+
+   composer create-project symfony/skeleton restaurants
+
+Install MongoDB Extension
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following commands to enter your project directory and install
+MongoDB:
+
+.. code-block:: bash
+
+   cd restaurants
+   pecl install mongodb
+
+Install PHP Driver and Doctrine ODM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following commands to install the MongoDB PHP driver and the
+MongoDB Doctrine ODM:
```

Comment:
Suggest to add a line that talks about the mongodb-odm-bundle and why we are installing it instead of directly installing mongodb-odm

Quoting the reason from github: "This bundle integrates the Doctrine MongoDB Object Document Mapper (ODM) library into Symfony so that you can persist and retrieve objects to and from MongoDB."

Also add a hyperlink to it - https://github.com/doctrine/DoctrineMongoDBBundle